Malt/Shaders/Node Utils/float.glsl

```diff
@@ -78,4 +78,23 @@ bool float_less_or_equal(float a, float b){ return a <= b; }
 /*META @meta: label=If Else;*/
 float float_if_else(bool condition, float if_true, float if_false){ return condition ? if_true : if_false; }
 
+/*META @meta: subcategory=Map Range; label=Float; 
+    @clamped: default=true;
+    @value: default = 0.5;
+    @from_min: default = 0.0;
+    @from_max: default = 1.0;
+    @to_min: default = 0.0;
+    @to_max: default = 1.0;
+*/
+float float_map_range(bool clamped, float value, float from_min, float from_max, float to_min, float to_max)
```

Comment:
Would these maybe make more sense as part of the Math nodes?

Reply:
I think, if the map range node wouldn't have multiple subcategories I would agree about b.
But the way it's now doesn't really make the setup any faster?

And we can add an enum for stepped/smooth/smoothstep in any case.